reana_job_controller/htcondorcern_job_manager.py

```diff
@@ -28,7 +28,7 @@
 class HTCondorJobManagerCERN(JobManager):
     """CERN HTCondor job management."""
 
-    MAX_JOB_RESTARTS = 3
+    MAX_RETRIES = 3
```

Comment:
Let's add some documentation stating how it is exactly being used. Also in https://github.com/reanahub/reana-job-controller/pull/176/files#diff-669c9f342cac1a455e5c652c40abf085R36